packages/neo-one-client-common/src/crypto.ts

```diff
@@ -575,6 +586,192 @@ const isSignatureContract = (script: Buffer) => script.length === 35 && script[0
 
 const isStandardContract = (script: Buffer) => isSignatureContract(script) || isMultiSigContract(script);
 
+const HARDENED_KEY_OFFSET = 0x80000000;
+const EXTENDED_KEY_BYTES = 78;
+
+// Need eyes, according to https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki#node-serialization this isn't special.
```

Comment:
Seems legit, go ahead and leave the comment with the url though so we know why this is here later.